hyperdrive.php

```
@@ -298,3 +303,28 @@ function get_deps_for_handle( $handle ) {
   $dep = get_dep_for_handle( $handle );
   return $dep->deps;
 }
+
+
+/**
+ * Checks if a value exists in a multidimensional array.
+ *
+ * @since Hyperdrive 1.0.0-beta
+ * @param string/array $needle The value(s) to search for.
+ * @param array $haystack The array to search.
+ * @param boolean $strict If set to TRUE also check the types of the needle in the haystack.
+ * @return boolean True if found, false otherwise.
+ */
+function in_multi_array($needle, $haystack, $strict = false) {
+  // The $haystack must be an array.
+  if ( !is_array($haystack) ) {
+    return false;
+  }
```

<del>incorrect</del> <ins>unexpected</ins>. The design philosophy being "fail fast and fail hard".